rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,65 @@
+"""
+Relationships with Foreign Keys, esp. relationship() and FK values
+Why do we need to establish columns for players in game?
+difference with many to one and one to many -- if you back_populate either relationship type,
+    does it matter which you use?
+
+"""
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import as_declarative
+from sqlalchemy.orm import relationship
+
+
+@as_declarative()
+class ModelBase(object):
+    def __tablename__(self):
+        return self.__class__.__name__.lower()
+
+
+class Game(ModelBase):
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'))
+    rounds_choice = Column(Integer)
+    rounds_decrement = Column(Integer)
+    current_round = Column(Integer)
+    winner = Column(String, ForeignKey('user.id'))
+
+    round = relationship('Round', back_populates='game')
+    user = relationship('User', back_populates='game', uselist=False)
+
+
+class Round(ModelBase):
+    id = Column(Integer, primary_key=True)
+    player_1_choice = Column(String, ForeignKey('choice.id'))
+    player_2_choice = Column(String, ForeignKey('choice.id'))
+    winner = Column(String, ForeignKey('user.score'))
+
+    game = relationship('Game', back_populates='round')
+    user = relationship('User', back_populate='user')
+    choice = relationship('Choice', back_populate='round')
+
+
+class User(ModelBase):
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    score = Column(Integer)
+    choices = Column(String, ForeignKey('choice.move'))
+
+    game = relationship('Game', back_populates='user')
```

Comment:
This would need to be a one-to-many relationship (one user, many games). So it should probably be named `games` (plural), and will also need a way specify foreign keys:
```python
games = relationship('Game', foreign_keys=['Game.player_1_id', 'Game.player_2_id'])
```

Disclaimer: I _believe_ that specifying multiple values in the `foreign_keys` list specifies "this _or_ that", but I could be mistaken.